lib/determinator/cache/active_support_memory_store.rb

```diff
@@ -0,0 +1,21 @@
+require 'active_support/cache'
+
+module Determinator
+  module Cache
+    class ActiveSupportMemoryStore
```

Comment:
This is useful for wrapping the call and giving it a feature name, but IMO no need to specify Memory store here. This is a generic `ActiveSupport::Cache` wrapper.

Reply:
Yeah, makes sense to me. I'll rename.

Reply:
Oh! I jsut remembered why I did that — to leave it as a generic name means someone might try and use a non-memory cache, eg. Redis, which will provide no benefit _at all. Still happy to rename, but I'm in two minds.

Reply:
Perhaps have a generic handler that defaults to using the memory store?

Reply:
Why the generalisation though? You're always going to want a cache, and never a cache that isn't in-memory — though someone might want to write a cache that doesn't use active support?

Reply:
You might be using a fork based server, in which case a FIleStore would be a lot better for you. You might be using memcahced, you might want to maintain a separate redis cache.

Very on board with MemoryStore as an sensible default, but I don't see a reason not to be flexible.

>though someone might want to write a cache that doesn't use active support

The `#fetch` pattern is super standard, i'd be surprised if there was a cache lib that didn't quack like that.